.github/workflows/trigger-e2e-tests.yml

```
@@ -55,7 +86,7 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)